packages/mermaid/src/diagrams/git/gitGraphRenderer.js

```diff
@@ -77,15 +79,20 @@ const drawCommits = (svg, commits, modifyGraph) => {
   const gLabels = svg.append('g').attr('class', 'commit-labels');
   let pos = 0;
 
+  if (dir === 'TB') {
```

Comment:
We need some tests covering that feature

Reply:
I am not familiar with testing. Can you please guide me on how to do it?

Reply:
There are 2 kinds of tests for the project. Unit tests (vitest) and integration tests or end-2-end tests (cypress). Unit tests check if a function works as expected, integration tests check what a user sees in browser. Please, skim through [testing section of contributing guidelines](https://mermaid.js.org/community/development.html#unit-tests), and feel free to ask if you have any questions.

Reply:
In the documentation, it's mentioned, we don't have to unit test renderers.

Reply:
I have added tests in cypress. Did all the tests which were done on horizontal branches.